packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+import SettingsPage from './SettingsPage.svelte';
+import EngineIcon from '../ui/EngineIcon.svelte';
+import EmptyScreen from '../ui/EmptyScreen.svelte';
+import { onMount } from 'svelte';
+import type { KubeContextUI } from '../kube/KubeContextUI';
+import { getKubeUIContexts } from '../kube/KubeContextUI';
+import Link from '../ui/Link.svelte';
+
+let contexts: KubeContextUI[] = [];
+
+$: contexts;
+
+onMount(async () => {
+  // Retrieve both the contexts and clusters
+  let k8sContexts = await window.kubernetesGetContexts();
+  let k8sClusters = await window.kubernetesGetClusters();
+
+  // Convert them to KubeContextUI so we can safely render them.
+  contexts = getKubeUIContexts(k8sContexts, k8sClusters);
+});
+</script>
+
+<SettingsPage title="Kubernetes Contexts">
+  <div class="h-full" role="table" aria-label="contexts">
+    <!-- Use KubernetesIcon in the future / not EngineIcon -->
+    <EmptyScreen
+      aria-label="no-resource-panel"
+      icon="{EngineIcon}"
+      title="No Kubernetes contexts found"
+      message="Check that ~/.local/config exists or KUBECONFIG environment variable has been set correctly."
```

Comment:
IMHO it's `$HOME/.kube/config` not .local

We also have a property in Podman Desktop so probably ask user to check the value of this property as well (KUBECONFIG won't work AFAIK)

Reply:
Updated! Good catch, it's suppose to be .kube/ not .local/